packages/server/src/games.ts

```diff
@@ -21,14 +21,15 @@ import {
 } from "./time-control/time-control";
 import { timeControlHandlerMap } from "./time-control/time-handler-map";
 import { Clock } from "./time-control/clock";
+import { updateRatings } from "./ratings";
 
 export function gamesCollection() {
   return getDb().db().collection("games");
 }
 
 /**
  * @param count number of games to return (default = 10, max = 100)
- * @param offset number of games to skip (default = 0)
+ * @param offset number of games to skip (default = 0)gamesCollection
```

Comment:
Is this addition intentional?

Reply:
It was not intentional, not sure where it came from. I'll go ahead and remove it.